<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yowsup</class>
 <widget class="QWidget" name="yowsup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>yowsup</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendSMSButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send SMS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateVersions">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update versions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="text">
    <string>After updating whatsapp, the application may stop working. Download the latest version
of  WhatsApp.apk from  https://www.whatsapp.com/android/ 
and place it in the program folder.  Then click the &quot;Update Versions&quot; button</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectJSONFolder">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select JSON folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="json_source">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>путь к json</string>
   </property>
  </widget>
  <widget class="QLabel" name="UpdateLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="send_message_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SEND MESSAGE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="config_source_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectConfig">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Config</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="recepientMessage">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>570</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recepientPhone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="spamButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>620</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Message</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
